licensing/src/main/java/io/crate/license/DecryptedLicenseData.java

```diff
@@ -53,23 +64,68 @@ String issuedTo() {
         return issuedTo;
     }
 
+    String signature() {
+        return (signature.isPresent() ? signature.get() : "");
```

Comment:
`Optional` would have a `orElse` method. (But see my other comment about usually not using it)